spec/models/user_spec.rb

```diff
@@ -0,0 +1,20 @@
+require 'rails_helper'
+
+RSpec.describe User, type: :model do
+  describe 'DB table' do
+    it { is_expected.to have_db_column :id }
```

Comment:
there is no need to test for `:id`, every model you create in rails have ids

Reply:
removed testing for id for user and product